src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -409,7 +410,7 @@ public void mergeAlignment(final File referenceFasta) {
                         final SAMRecord matePrimary = isRead1 ? r2Primary : r1Primary;
 
                         for (final SAMRecord supp : supplementals) {
-                            final SAMRecord out = clone(sourceRec);
+                            final SAMRecord out = sourceRec.deepCopy();
```

Comment:
@nh13 You're right, the SAM spec says that a chimeric alignment is a set of linear alignments and linear alignments must be on the same strand. If we assume that, I can change it back to clone for supplementals.

@yfarjoun makeReadUnmapped() isn't the only problem, it's also in AbstractAlignmentMerger::setValuesFromAlignment(). This means that we need a deep copy before calling transferAlignmentInfoToPairedRead() or transferAlignmentInfoToFragment(), the way it is written now. Additionally, you won't know you need a deep copy until you've seen every secondary alignment that came from the original read.